redis/src/main/scala/zio/redis/options/SortedSets.scala

```diff
@@ -101,4 +101,7 @@ trait SortedSets {
   }
 
   type WithScores = WithScores.type
+
+  sealed case class Weights(weights: Seq[Long])
```

Comment:
This should not be needed at all. You can still use `DoubleInput` (not longs) and prepend `WEIGHTS` during command encoding.